src/ColorScale.js

```diff
@@ -201,8 +223,17 @@ export default class ColorScale extends BaseClass {
             .map(d => quantile(allValues, d));
         }
         else {
-          const step = (domain[1] - domain[0]) / (colors.length - 1);
-          buckets = range(domain[0], domain[1] + step / 2, step);
+          if ((this._scale === "linear" || this._scale === "log") && diverging && this._color && this._centered) {
```

Comment:
if you remove `(this._scale === "linear" || this._scale === "log") &&` entirely, it makes this new logic work with `scale: "buckets"` 😏